src/TestFramework/PhpUnit/CommandLine/ArgumentsAndOptionsBuilder.php

```diff
@@ -64,4 +69,36 @@ public function build(string $configPath, string $extraOptions): array
 
         return $options;
     }
+
+    /**
+     * @param TestLocation[] $tests
+     */
+    public function buildForMutant(string $configPath, string $extraOptions, array $tests): array
+    {
+        $options = $this->buildForInitialTestsRun($configPath, $extraOptions);
+
+        if (count($tests) > 0) {
```

Comment:
I don't think we can take just the number of tests, and test suite time wouldn't necessary give the whole picture. I'd say median test time multiplied on the number of tests will give a better estimate to the time measure because median naturally filters outliers. Or we can go full with [three standard deviations of the mean](https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule), which should give possibly even better estimate.